src/Tribe/Template/Month.php

```diff
@@ -401,6 +401,7 @@ protected function set_args( $args = array() ) {
 					'eventDisplay' => 'month',
 					'eventDate'    => is_array( $_POST['eventDate'] ) ? Tribe__Utils__Array::get( $_POST, array( 'eventDate', 0 ) ) : $_POST['eventDate'],
 					'post_status'  => $post_status,
+					'tribe_suppress_query_filters' => false
```

Comment:
:no_entry_sign: Each line in an array declaration must end in a comma

<sub>hash: 36ff7e4f8717c6acbb5730b77364179c</sub>